packages/next/src/server/lib/patch-fetch.ts

```diff
@@ -100,6 +100,13 @@ export function patchFetch({
     const fetchStart = Date.now()
     const method = init?.method?.toUpperCase() || 'GET'
 
+    // Do create a new span trace for internal fetches in the
+    // non-verbose mode.
+    const isInternal = (init?.next as any)?.internal === true
+    if (isInternal && process.env.NEXT_OTEL_VERBOSE !== '1') {
```

Comment:
@timneutkens I changed the approach here a bit. Instead of checking the verbosity here, I renamed the internal fetch event, since it's for things like IPC.